packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -90,6 +93,17 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
       ));
     }
 
+    // Python dependencies installed
+    if (hasPythonSixModule) {
+      pythonStatus = ValidationType.installed;
+    } else {
+      pythonStatus = ValidationType.missing;
+      messages.add(new ValidationMessage.error(
+        'Python installation missing module "six".\n'
+        'Install via \'sudo easy_install six\'.'
```

Comment:
Ya, it's hard to make that assumption. They can be missing the module for all sorts of weird reasons. 

Some soft-ish language like you had is probably fine. Like your thing is missing, we don't presume to know why but if you wanted to install it, you can do it like this: